apps/aehttp/test/aehttp_sc_SUITE.erl

```diff
@@ -730,7 +730,7 @@ channel_update(#{initiator := IConnPid, responder :=RConnPid},
 
     %% validate it is co-signed
     {ok, Trees} = rpc(aec_chain, get_top_state, []),
-    ok = rpc(aetx_sign, verify, [SignedStateTx, Trees]), % RPC because of DB
+    ok = rpc(aetx_sign, verify, [SignedStateTx, Trees, 1]), % RPC because of DB
```

Comment:
is 1 a valid environment? Yes, verify/3 changed to take height as 3rd argument. Ok.